atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -2050,12 +2058,13 @@ private void commitWrites(TransactionService transactionService) {
             // timed.
             traced(
                     "preCommitLockCheck",
-                    () -> preCommitRequirementValidator.throwIfImmutableTsOrCommitLocksExpired(commitLocksToken));
+                    () -> preCommitRequirementValidator.throwIfImmutableTsOrCommitLocksExpired(
```

Comment:
Checked that `preCommitRequirementValidator.throwIfImmutableTsOrCommitLocksExpired` also validates the immutableTs lock is valid, so no change in behavior.